docs/analytics.rst

```diff
@@ -13,16 +13,16 @@ Google Tag Manager (GTM)
 
 Bedrock uses Google Tag Manager (GTM) to manage and organize its Google Analytics solution.
 
-GTM is a tag management system that allows for easy implementation of Google Analytics tags and other 3rd party marketing tags in a nice GUI experience. Tags can be added, updated, or removed directly from the GUI. GTM allows for a `one source of truth` approach to managing an analytics solution in that all analytics tracking can be inside GTM.
+:abbr:`GTM (Google Tag Manager)` is a tag management system that allows for easy implementation of Google Analytics tags and other 3rd party marketing tags in a nice GUI experience. Tags can be added, updated, or removed directly from the GUI. :abbr:`GTM (Google Tag Manager)` allows for a `one source of truth` approach to managing an analytics solution in that all analytics tracking can be inside :abbr:`GTM (Google Tag Manager)`.
```

Comment:
This might be excessive. I can update to only use `abbr` for the first instance in a paragraph. Or, at least, the first instance if used multiple times in the same sentence.

Reply:
Yeah, when it's in the same paragraph maybe we should only do the first one.

Also, should we define "GUI" or is that one ubiquitous enough? (same question for CLI, if we use that anywhere)

Reply:
GUI I would say is well known, CLI less so. I can add CLI.